indico/modules/events/persons/templates/management/_person_list_row.html

```diff
@@ -26,7 +26,14 @@
     {% endif %}
     <td class="i-table name-column" data-searchable="{{ person.display_full_name|lower }}">
         {{ person.display_full_name }}
-        {% if person.is_untrusted %}
+        {% if 'no_roles' in person_data.roles %}
+            {% set no_roles_tooltip -%}
+                {%- trans -%}
+                    This person currently has no ties to the event.
+                {%- endtrans -%}
+            {%- endset %}
```

Comment:
I think this string is short enough (and it has no linebreaks) to just put it directly in the `title="..."`